docs/_snippets/features/horizontal-rule.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* globals window, document, console */
+
+import ClassicEditor from '@ckeditor/ckeditor5-build-classic/src/ckeditor';
```

Comment:
Missing in package.json AFAICS.

Reply:
AFAIK we don't add "docs deps" to pkg.json.